pkg/strategy/bollmaker/strategy.go

```diff
@@ -435,14 +445,20 @@ func (s *Strategy) placeOrders(ctx context.Context, midPrice fixedpoint.Value, k
 		}
 	}
 
+	if !s.shouldBuy {
```

Comment:
Not shouldBuy != Should not buy

Maybe renane this variable to shouldNotBuy?

Reply:
shouldBy is set to true by default, would like to keep it simple to not to put a reversed logic in the variable ^^||